packages/postcss-reduce-initial/src/script/__tests__/io.js

```diff
@@ -0,0 +1,86 @@
+import nock from 'nock';
+import { handleError, toJSONString, write, generate } from '../lib/io.mjs';
+import testData from './sampleProperties.json';
+
+describe('toJSONString', () => {
+  const rawData = {
+    foo: 'bar',
+    'baz-qux': 'quux',
+    quuz: 'corge',
+    garply: 'waldo',
+    'fred-plugh': 'xyzzy-thud',
+  };
+
+  test('should have correct data type (string)', () => {
+    expect(typeof toJSONString(rawData)).toBe('string');
+  });
+
+  test('should produce parsable JSON', () => {
+    expect(JSON.parse(toJSONString(rawData))).toEqual(rawData);
```

Comment:
My thought with this test was that... if we...

1. Validate that `toJSONString` produces parsable JSON
2. Assume that writeFile does it's thing (not our responsibility to test)
3. In the assertion on line 43 verify that `writeFile` was called with the data this function produces... 

Then we can reasonably assume that valid JSON would be written to the file. 

And indirectly that it would be transpilable, and usable by the plugin